### DISCUSSION TITLE
new version of github codespace

### DISCUSSION THREAD

**Discussion by Ahmadshafiq327** [community/community#125650] [Codespaces]
### Select Topic Area

Bug

### Body

vs code in extension github codespace new version updated.
but github codespace dont open in vs code..........error of update new version.
![new version](https://github.com/community/community/assets/169985651/87866a90-8d46-4723-80ff-bbe4fee65ab9)

**Comment by Ahmadshafiq327** [comment]
please solve this problem

**Comment by UjjwalSaini07** [comment]
Hii I try to solve ur Problem
_**Website Link - https://code.visualstudio.com/docs/remote/codespaces
Official GitHub Codespaces Link - https://github.com/features/codespaces**_
Visit to these website to overcome your Problem

Also Use this Original Extension Of Github Codespace:

![image](https://github.com/community/community/assets/73696489/db08d170-044a-403a-aefa-728045f2ded4)

**_And Mind it Before using that that extension Refer to that Repository: https://github.com/codespaces_**

### **_If U Like this answer So please make as correct answer hai show thumbs Up_**

**Comment by Ahmadshafiq327** [comment]
Till now don't resolve my problem in visual studio new version of GitHub
codespace

**Reply by UjjwalSaini07** [reply]
what do u means that

**Reply by Ahmad-shafiq327** [reply]
I mean
New version of GitHub codespace.... don't updated in my PC

**Comment by Ahmadshafiq327** [comment]
I mean...
New version of GitHub codespace don't updated in my PC
I don't understand

**Comment by UjjwalSaini07** [comment]
**### _For Visual Studio Code:_**

**_1. **_Open Visual Studio Code._**_**

**_2. _**Go to Extensions:**__**

- Click on the Extensions icon in the Activity Bar on the side of the window.
- Alternatively, you can use the keyboard shortcut Ctrl+Shift+X (Windows/Linux) or Cmd+Shift+X (macOS).

**_3. Check for Updates:_**

-  In the Extensions view, you should see a list of installed extensions.
- If any of these extensions have updates available, there will be an update button next to them. Click on this button to update the extension.
-  You can also click the gear icon (⚙️) next to an extension and select "Check for Updates" to manually check for updates.

**_4. Update All Extensions:_**
- To update all extensions at once, click on the three dots (More Actions) at the top of the Extensions view and select "Update All Extensions."

_**5. Reload/Restart VS Code:**_
-  After updating the extensions, you may need to reload VS Code for the changes to take effect. You can do this by clicking on the reload button that appears after an extension is updated or by restarting VS Code.


**### _For GitHub Codespaces:_**

_**1. Open your Codespace.**_

_**2. Go to Extensions:**_

- Click on the Extensions icon in the Activity Bar on the side of the window.

_**3. Follow the same steps as above for checking and updating extensions**_


**### _Update via Command Line:_**

_**1. Open your terminal.**_

_**2. List installed extensions:**_

`code --list-extensions`

_**3. Update an extension:**_

`code --install-extension <extension-id> --force`

Replace <extension-id> with the ID of the extension you want to update.

**Comment by Ahmad-shafiq327** [comment]
first of all.....
i have already setting in visual studio code auto update all extention.
but new version of github codespace don't show me in extension.

**Comment by UjjwalSaini07** [comment]
In my laptop it works normally so delete that extension and again download it

**Reply by UjjwalSaini07** [reply]
Download This `https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces`

**Comment by Ahmadshafiq327** [comment]
what does mean
